src/Utilitywarehouse.OpsEndpoints/Owin/OpsEndpointsMiddleware.cs

```diff
@@ -21,14 +21,16 @@ public class OpsEndpointsMiddleware
         private readonly RequestDelegate _next;
         private readonly JsonSerializer _serializer;
         private readonly Dictionary<string, HttpHandler> _handlers;
+        private readonly JsonSerializerSettings _serializerSettings;
 
         public OpsEndpointsMiddleware(RequestDelegate next, OpsEndpointsMiddlewareOptions options)
         {
             _next = next;
             _handlers = ConfigureHandlers(options);
-            var serializer = new JsonSerializer {ContractResolver = new CamelCasePropertyNamesContractResolver()};
-            serializer.Converters.Add(new StringEnumConverter());
-            _serializer = serializer;
+            _serializerSettings = new JsonSerializerSettings
+            {
+                Formatting = Formatting.Indented
```

Comment:
Bear in mind that we want this to return quickly, and any overhead should be avoided.